### DISCUSSION TITLE
Tabbed result panes separated

### DISCUSSION THREAD

**Discussion by ghost** [dbeaver/dbeaver#11983] [Q&A]
Not sure this is a bug. I don't how but somehow the Results tab has split into separate panes. I cannot find a way to put them all back into the one tabbed 'page' like it used to be.

![image](https://user-images.githubusercontent.com/72906479/113502978-867bc200-9572-11eb-9775-5b38a9b80070.png)

Please can someone advise how to get the panes into one tabbed result window?

Thanks

Windows 10 19041
 DBeaver 21.0.1
MS SQL Server 2019
Microsoft JDBC Driver for SQL Server
Straight TCPIP connection

**Comment by LonwoLonwo** [comment]
Hello @PaulSlater25 this is separate output tab https://github.com/dbeaver/dbeaver/issues/10683

![104336564-98b30a80-5505-11eb-91f8-7af1d2145704](https://user-images.githubusercontent.com/45152336/113511735-18cb9800-956a-11eb-9c4e-22b198b66c0b.png)
You can turn it off in preferences->Editors->SqlEditor
